pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -633,6 +638,34 @@ func (s *HttpSettings) Validate() error {
 		}
 	}
 
+	for _, reg := range s.FailIfBodyMatchesRegexp {
```

Comment:
CI seems to be complaining about the increased cyclomatic complexity, probably because it sees to many `for` loops and gets scared. Perhaps we can `//nolint` this one out, or alternatively `append` all regexes together and return the same error in case any group is wrong.

Reply:
I'm inclined to `//nolint:gocyclo` this one since the specific error message seems nice to have.

I'll tinker around to see if there's an easy way to do it in a loop (maybe using a `map[error][]string`) but having the mix of `[]string` and `[]HeaderMatch` might complicate that.

Reply:
That's a good point. I'd probably `nolint` it away instead of building a complex workaround 😃